views.js

```diff
@@ -70,14 +78,35 @@ function renderTitlebar (state, width) {
   ]
 }
 
+function renderCabals (state, width, height) {
+  return state.cabals
+    .map(function (cabal, idx) {
+      var key = cabal.key
+      var keyTruncated = key.substring(0, width - 1)
+      if (state.cabal.key === key) {
+        if (state.selectedWindowPane === 'cabals') {
+          return '>' + chalk.bgBlue.bold(keyTruncated)
```

Comment:
nice sigil :)

Reply:
Actually, I'm going to keep it at > for now. (➤ ends too close to the next character to look good)